safehttp/incoming_request.go

```diff
@@ -47,11 +50,15 @@ func NewIncomingRequest(req *http.Request) *IncomingRequest {
 	if req == nil {
 		return nil
 	}
+	req = req.WithContext(context.WithValue(req.Context(),
+		flightValuesCtxKey{}, &flightValues{m: map[interface{}]interface{}{}}))
```

Comment:
make(map[interface{}]interface{}) might be a little easier to read here than a literal

Reply:
Done in 0ce62ea.